monai/data/iterable_dataset.py

```diff
@@ -181,7 +182,8 @@ class CSVIterableDataset(IterableDataset):
 
     def __init__(
         self,
-        filename: Union[str, Sequence[str]],
+        filename: Optional[Union[str, Sequence[str]]] = None,
+        iter: Optional[Union[Iterable, Sequence[Iterable]]] = None,
```

Comment:
I think these could be merged into a `src` option?

Reply:
OK, I have merged in the latest commit.
Could you please help review it again?

Thanks in advance.